packages/app/cypress/e2e/runner/runner.ui.cy.ts

```diff
@@ -157,19 +157,21 @@ describe('src/cypress/runner', () => {
 
     it('renders spec name and runtime in header', () => {
       loadSpec({
-        fileName: 'simple-cy-assert.runner.cy.js',
+        filePath: 'runner/simple-cy-assert.runner.cy.js',
         passCount: 1,
         failCount: 0,
         hasPreferredIde: true,
       })
 
-      cy.intercept('mutation-SpecRunnerOpenMode_OpenFileInIDE', { data: { 'openFileInIDE': true } }).as('OpenIDE')
+      cy.withCtx((ctx, o) => {
+        ctx.actions.file.openFile = o.sinon.stub()
+      })
```

Comment:
The helper is not using `__incorrectlyVisitAppWithIntercept` anymore, so replaced the intercept with appropriate stubbing.